helpers/utils.js

```diff
@@ -106,6 +108,12 @@ async function newPage(context) {
     return page;
 }
 
+async function newContext(browser, options = {}) {
+    const context = await browser.createIncognitoBrowserContext(options);
+    limitContext.incContextCounter();
+    return context;
```

Comment:
Suggestion: check context limit here and raise exception if it is exhausted, similar to context/page not found exceptions. Then check it in middleware and send 429.